From: Arturo Borrero Gonzalez <arturo@debian.org>
Subject: Drop XML tests
 This patch drops XML tests, since we don't support XML parsing.
Forwarded: not-needed
Last-Update: 2016-10-10

--- a/tests/test-script.sh
+++ b/tests/test-script.sh
@@ -20,5 +20,4 @@
 ./nft-rule-test
 ./nft-set-test
 ./nft-table-test
-./nft-parsing-test -d xmlfiles
 ./nft-parsing-test -d jsonfiles
